addons/website/static/src/js/utils.js

```diff
@@ -239,6 +239,32 @@ function websiteDomain(self) {
     return ['|', ['website_id', '=', false], ['website_id', '=', websiteID]];
 }
 
+/**
+ * Checks if the 2 given URLs originates from the same domain.
+ * It will consider naked URL and `www` URL originating from the same domain.
+ * It will consider `https` URL `http` URL originating from the same domain.
+ *
+ * @param {string} url1
+ * @param {string} url2
+ * @returns {Boolean}
+ */
+function areSameDomain(url1, url2) {
+    try {
+        url1 = new URL(url1).host;
+        url2 = new URL(url2).host;
+    } catch (_err) {
```

Comment:
As you prefer, `} catch (_` is the most commonly used way of writing it, even if there a very few occurences of `} catch {` in the code.
Might be because linters are rejecting this form
![image](https://user-images.githubusercontent.com/30048408/205355458-5c455901-9611-4483-a559-f996d0b90d4f.png)

> (I wonder if runbot will even let you merge

Yes, it's actually the reason I replaced `err` by `_err` https://runbot.odoo.com/runbot/build/21555501

Reply:
Your linter is outdated then ;) It's not what I prefer, `(_err)` will make the runbot red in master: https://github.com/odoo/odoo/commit/4f24e34b39c35ec212e906bb18348ba32136e2c7 (if you believe the commit message).

Reply:
It's clearly not as my runbot was green :p (but I adapted already anyway)

Reply:
Note that it broke the whole js linter, nothing else is linter in the whole file :o

Reply:
> It's clearly not as my runbot was green :p (but I adapted already anyway)

Because this PR is for 16.0, not master. It would be broken during the forward-port :shrug:

Reply:
Can't make eslint work with option catch binding from ES2019, I'll check later, whatever :(